src/Core/Parser/BooleanParser.php

```diff
@@ -6,8 +6,6 @@
  * See LICENSE.txt that was shipped with this package.
  */
 
-use TYPO3Fluid\Fluid\Core\Parser\Exception as ParserException;
```

Comment:
This one still is in use in `parseTermToken()`

Reply:
Agreed, now solved